pkg/mhooks/event.go

```diff
@@ -0,0 +1,47 @@
+package mhooks
```

Comment:
Nit: I don't know what "mhooks" means. Perhaps "moovhooks" is clearer?

Reply:
Yeah, can't say I love `mhooks` either.. i like `moovhooks` more too. @InfernoJJ did mentioned this naming since he plans to rename `moov` package to `mapi`